bin/morgue.js

```diff
@@ -4950,14 +4958,21 @@ function coronerList(argv, config) {
         console.log(pp);
         return;
       }
+      // determine type of response - if json values contain only one element
```

Comment:
I believe these comments are out of date: the actual logic is if the user specified select. The code here is clear enough that I'd just drop the comments entirely, unless I'm missing something about the logic.

Reply:
We still need them to determine if we should create empty csv file or not. There is no need to create writeStream and care at all about csv integration if there is no data or we downloaded aggregated data.

Reply:
On the other hand ... if we will later extend csv integration with aggregation results/pretty prints/whatever we want - it will be usefull to not care about different parts of code.

Reply:
@camlorn previously we added requirement - we should be able to export csv data if filters are available (so only for raw data without aggregation). I will leave it and get more feedback about customer needs. I added console.log to tell user what happened.

Reply:
My issue isn't the logic, it's that the comment claims that it's using the json when it appears to be using the command line flags the user passed in instead.  I think we should be able to write CSVs for aggregation since if you add `--factor` you can get lots and lots of aggregation rows back, so I don't see any particular need to distinguish these cases, but don't need that today for my sign-off.